src/unstract/sdk/vector_db.py

```diff
@@ -152,18 +165,31 @@ def add(
             nodes=nodes,
         )
 
-    @deprecated("Use the new class VectorDB")
-    def get_vector_db(
-        self, adapter_instance_id: str, embedding_dimension: int
-    ) -> Union[BasePydanticVectorStore, VectorStore]:
-        self._embedding_dimension = embedding_dimension
-        return self._vector_db_instance
-
     def close(self, **kwargs):
         if not self.vector_db_adapter_class:
             raise VectorDBError("Vector DB is not initialised properly")
         self.vector_db_adapter_class.close()
 
+    def get_class_name(self) -> str:
+        """Gets the class name of the Llama Index Vector DB.
+
+        Args:
+            NA
+
+            Returns:
+                Class name
+        """
+        return self._vector_db_instance.class_name()
+
+    @deprecated("Deprecated class and method. Use VectorDB instead of ToolVectorDB")
```

Comment:
@gaya3-zipstack `deprecated` library will throw warning that it is deprecated, we do not need to mention again.

In that case, let's change message to `Use VectorDB instead of ToolVectorDB.`.

Reply:
Done